### PR TITLE
doc: dfu update image ver instruction

### PR DESCRIPTION
**Pull request by kittydepa**
[NCSDK-26586](https://nordicsemi.atlassian.net/browse/NCSDK-25919)
As reported by SH in support:
Update image version instruction for MCUboot section

[NCSDK-26586]: https://nordicsemi.atlassian.net/browse/NCSDK-26586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by nvlsianpu — APPROVED** [review]

File: doc/nrf/config_and_build/dfu/dfu_image_versions.rst [hunk 1]
```diff
@@ -36,7 +36,9 @@ This option can refer to two different things:
 Configuring image version with MCUboot
 **************************************
 
-To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), add the version string to the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
+To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), it is recommended to use the :file:`VERSION` file that contains the version information for the application image.
+
+Alternatively, you can also add the version string to the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
```
Comment (nvlsianpu) [inline doc/nrf/config_and_build/dfu/dfu_image_versions.rst:41]: which allows to configure the version value for each application build instance independently.

Comment (kittydepa) [reply doc/nrf/config_and_build/dfu/dfu_image_versions.rst:42]: Added

File: doc/nrf/config_and_build/dfu/dfu_image_versions.rst [hunk 2]
```diff
@@ -36,7 +36,10 @@ This option can refer to two different things:
 Configuring image version with MCUboot
 **************************************
 
-To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), add the version string to the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` option for the application:
+To assign a semantic version number to your application when you have opted for booting application directly by the MCUboot bootloader (that is, if you have opted for either :ref:`ug_bootloader_adding_immutable_mcuboot` or :ref:`ug_bootloader_adding_upgradable_mcuboot`), it is recommended to use the :file:`VERSION` file that contains the version information for the application image.
```
Comment (hellesvik-nordic) [inline doc/nrf/config_and_build/dfu/dfu_image_versions.rst:39]: I suggest that we link to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/build/version/index.html

File: doc/nrf/config_and_build/dfu/dfu_image_versions.rst [hunk 3]
```diff
@@ -36,7 +36,10 @@ This option can refer to two different things:
...
+Using a :file:`VERSION` file allows you to configure the version value for each application build instance independently.
```
[the remark this answers is not in the document]
Comment (kittydepa) [reply doc/nrf/config_and_build/dfu/dfu_image_versions.rst:40]: Agree, will apply

**Review by hellesvik-nordic — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by annwoj — APPROVED** [review]

**Pull request merged**